backend/src/packages/chaiNNer_standard/image_dimension/utility/get_bounding_box.py

```diff
@@ -0,0 +1,57 @@
+from __future__ import annotations
+
+from typing import Tuple
+
+import numpy as np
+
+from nodes.properties.inputs import ImageInput, SliderInput
+from nodes.properties.outputs import NumberOutput
+from nodes.utils.utils import get_h_w_c
+
+from .. import utility_group
+
+
+@utility_group.register(
+    schema_id="chainner:image:get_bbox",
+    name="Get Bounding Box",
+    description="Gets a bounding box (X, Y, Height, and Width) of the white area of a mask.",
+    icon="BsRulers",
```

Comment:
Ruler? Maybe there's the better icon for it... Not sure what though.

Reply:
Where do the icons come from? I may be able to find a better one.

Reply:
Nevermind, I found the React Icons. I'll try and find one

Reply:
![image](https://github.com/chaiNNer-org/chaiNNer/assets/54873330/9cd61b13-3bc7-4120-8a58-676a9eb01c20)
How does this look?

Reply:
Sorry for not telling you sooner. We use React Icons.

Yes, that icon looks good!